pkg/k8s/constants.go

```diff
@@ -77,8 +77,9 @@ const (
 	ManagementConflictPreventionPolicyResource = "resource"
 
 	// State into spec annotation values
-	StateMergeIntoSpec = "merge"
-	StateAbsentInSpec  = "absent"
+	StateMergeIntoSpec               = "merge"
```

Comment:
Nit: More go-idiomatic would be `StateIntoSpecMerge`.  Sometimes with a `type StateIntoSpec string`.  Not important to fix here though, as it is pre-existing.